frontend/src/components/Documentation.vue

```diff
@@ -368,10 +364,10 @@
                 ). To clear the RNA levels, select the
                 <i>None</i>
                 option in the drop down list. RNA levels are available for both 2D and 3D Map
-                Viewer. The data is obtained from version 18 of the Protein Atlas with the units in
-                log
-                <sub>2</sub>
-                (TPM+1) associated with a gradient colorbar.
+                Viewer. The gene expression levels used are obtained from the file
```

Comment:
Maybe add that this file is from protein atlas?

Reply:
"For Human-GEM, the gene expression levels for genes from The Human Protein Atlas can be loaded" is two sentences before this one. I think it's ok to only mention it once, is it ok with you as well?